source/runtime.cpp

```diff
@@ -601,40 +672,51 @@ void reshade::runtime::load_effects()
 #endif
 	_last_shader_reload_successful = true;
 
+	// Copy is required because load_effect is runs asynchronously.
+	ini_file &preset = ini_file::load_cache(_current_preset_path);
```

Comment:
Comment says a copy is required, but a reference is used?

Reply:
Sorry, this comment is wrong now.